nixos/modules/services/networking/smokeping.nix

```diff
@@ -340,9 +340,9 @@ in
       };
       preStart = ''
         mkdir -m 0755 -p ${smokepingHome}/cache ${smokepingHome}/data
-        ln -sf ${cfg.package}/htdocs/css ${smokepingHome}/css
-        ln -sf ${cfg.package}/htdocs/js ${smokepingHome}/js
-        ln -sf ${cgiHome} ${smokepingHome}/smokeping.fcgi
+        rm -f ${smokepingHome}/css && ln -sf ${cfg.package}/htdocs/css ${smokepingHome}/css
```

Comment:
`ln -sf` is supposed to overwrite the target location, was this broken without it?

Reply:
Ah, I see in the description, I'm uncertain this is the right fix though.

Reply:
I was sure this pull request was going to query :)

For my part, I do not see any permission problem (see the screenshot)
![2023-04-24_17-33](https://user-images.githubusercontent.com/2806307/234045429-9bdf255e-2521-4407-8edc-a7c8c56ee608.png)

Reply:
I finally found the problem, the symlink pointed to file instead of directory (See the resolution problem in the below screenshot).

![2023-04-25_23-13](https://user-images.githubusercontent.com/2806307/234406349-64e4537b-89e9-419c-bb08-030571e2edae.png)